js/index.js

```diff
@@ -0,0 +1,119 @@
+let books = JSON.parse(localStorage.getItem('books'));
+
+//  Error Message
+function errorMsg(message) {
+  document.querySelector('.warning').textContent = message;
+  setTimeout(() => {
+    document.querySelector('.warning').textContent = '';
+  }, 2000);
+}
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
```

Comment:
- [OPTIONAL] You could use a library that automatically generates for you a unique id such as UUID directly in the constructor so that you don't have to pass in the id as a parameter. Whats your take? 🤔